src/library-authoring/data/api.ts

```diff
@@ -702,10 +702,10 @@ export async function restoreContainer(containerId: string) {
 /**
  * Fetch a library container's children's metadata.
  */
-export async function getLibraryContainerChildren(
+export async function getLibraryContainerChildren<T>(
```

Comment:
Instead of `T` can you call this `ChildType` or something more obvious, and make it default to `LibraryBlockMetadata[] | Container[]` ?

```suggestion
export async function getLibraryContainerChildren<ChildType = LibraryBlockMetadata | Container>(
```

Same with `useContainerChildren` - just looking at the hook it's not obvious what the `T` type parameter is for, and it should have a sensible default type.

Reply:
That looks better. Thanks!